javascript-local-storage/data.js

```diff
@@ -0,0 +1,11 @@
+/* exported todos */
+
+let todos = [];
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+todos = JSON.parse(previousTodosJSON);
```

Comment:
You should have an `if` statement here, checking if `previousTodosJSON` is not null